edge-server/anthos/node-setup-common.sh.tmpl

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env sh
 
+KUBECONFIG="$SETUP_DIR/bmctl-workspace/$ANTHOS_MEMBERSHIP_NAME/$ANTHOS_MEMBERSHIP_NAME-kubeconfig"
```

Comment:
Can you add `{` and `}` to delimit vars?

Reply:
updated